src/System.Threading.AccessControl/tests/System.Threading.AccessControl.Tests.csproj

```diff
@@ -16,7 +16,7 @@
     <ProjectReference Include="..\src\System.Threading.AccessControl.csproj">
       <Project>{E3ED83FD-3015-4BD8-A1B8-6294986E6CFA}</Project>
       <Name>System.Threading.AccessControl</Name>
-      <OSGroup>$(TargetOS)</OSGroup>
+      <OSGroup>$(InputOSGroup)</OSGroup>
```

Comment:
What happens with this if you build this project in isolation and not from the root build?